src/checksum.rs

```diff
@@ -70,6 +72,15 @@ impl Checksum {
     }
 }
 
+impl Debug for Checksum {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        // Not a particularly informative Debug impl, but allows
+        // `Checksum` to be embedded in other structs that derive
+        // `Debug`.
+        f.debug_struct("Checksum").finish_non_exhaustive()
```

Comment:
And you don't want to do a digest.clone().finalize() or something for performance reasons? Or you don't actually think it'd be useful?
I looked at the integration branch, but didn't actually track down all the places this might get surfaced.

Reply:
I don't think so, for two reasons:
1. `finalize` wouldn't show you the actual internal state: https://docs.rs/crc/latest/src/crc/crc32.rs.html#18
2. In practice I don't think it would be useful to look at the value in a failing test, if there was a real checksum issue you'd probably need to insert various hex prints to find the problem.

This output should never make it out into the public API; Checksum is not pub, and neither is Inode (where it will be a field type in a later commit).